nodejs/week1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express();
+
+const meals = require ("../data/meals.json");
+const reviews = require ("../data/reviews.json");
+const cheapMeal = meals.filter(itemMeal => { 

```

Comment:
`filter` function has to return boolean. Currently, it returns smth explicitly only if `itemMeal.price < 50`. Otherwise, it will return undefined implicitly. Long story short - it's better to just return `itemMeal.price < 50` without any if 😉

I left a similar comment here:

https://github.com/Mekdes1/hyf-homework/pull/20#discussion_r382204955

Reply:
This pattern keeps repeating farther in your homework. It would be nice to fix.